### DISCUSSION TITLE
Still only Front Camera issue

### DISCUSSION THREAD

**Discussion by hatobel** [community/community#66866] [GitHub Education]
### Hi everyone,

I’ve tried applying to your Student program several times. Back in June I’ve made it work by using two phones, one to show my digital ISIC and one to snap a picture of it.
Now I can’t seem to get it working. I believe the main reason is, that no matter what phone I use, I can’t seem to switch to the back camera. Neither Android nor Apple phones (iPhone 13) will switch cameras.
Because of that I the quality of the picture is probably to bad for the verification bot.
I’ve also tried using the upload picture button but that doesn’t seem to work, neither with my School ID nor with my ISIC, although that worked in June.
I've seen similar discussions but those don't seem to get properly handled?

**Comment by gabrielqueeiroz** [comment]
I'm having the same issue, I've the screenshot that I can't attach, and when I take a picture it only accepts the front camera, hope they fix it soon.

**Reply by tushar3-0x** [reply]
Damn serious issue

**Reply by asdfghjkllbev** [reply]
所有我的鏡頭還是開著的？還有Wi-Fi怎麼跟改回來，電腦是公司的，我不能跟改，等等用出問題，我早就沒電腦了，只有手機

**Comment by tushar3-0x** [comment]
Can you please say that in English, I can't understand your language

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
